src/options/quantifiers_options

```diff
@@ -340,7 +340,7 @@ option cbqiBvInterleaveValue --cbqi-bv-interleave-value bool :read-write :defaul
 option cbqiBvSlackIneq --cbqi-bv-slack-ineq bool :read-write :default true
   use model slack values when solving inequalities with word-level inversion approach
 option cbqiBvInvInDisEq --cbqi-bv-inv-in-dis-eq bool :read-write :default false
-  let bv inverter handle (un)signed less than nodes
+  let bv inverter handle inequalities and disequalities
```

Comment:
This will disappear when you merge master (the option is subsumed by cbqiBvIneq=MODE).

Reply:
Ik, didn't merge master in yet :)